client/src/pages/HomePage.js

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+import LoginForm from '../components/LoginForm'
+import ProfileForm from '../components/ProfileForm'
+
+HomePage.propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+  onLogin: PropTypes.func.isRequired,
+}
+
+export default function HomePage({ onSubmit, onLogin }) {
+  const [entryForm, setEntryForm] = useState(false)
+  return (
+    <>
+      {!entryForm && (
+        <Wrapper>
+          <LoginForm onLogin={onLogin} />
+          <span
+            onClick={() => setEntryForm(!entryForm)}
+            data-testid="modusForm"
+          >
+            or registrate
+          </span>
+        </Wrapper>
+      )}
+      {entryForm && (
+        <Wrapper>
+          <ProfileForm onSubmit={onSubmit} />
+          <span
+            onClick={() => setEntryForm(!entryForm)}
+            data-testid="modusForm"
+          >
+            or log in
+          </span>
+        </Wrapper>
+      )}
```

Comment:
This is not easy to read. So you either want the user to register or log in, right? `entryForm` is not a good name for a bool. How about `[isRegistered, setIsRegistered]`?
```suggestion
 <Wrapper>
      {isRegistered ? (
          <LoginForm onLogin={onLogin} />
          <span
            onClick={() => setEntryForm(!entryForm)}
            data-testid="modusForm"
          >
            or register
          </span>
      ) : (
          <ProfileForm onSubmit={onSubmit} />
          <span
            onClick={() => setEntryForm(!entryForm)}
            data-testid="modusForm"
          >
            or log in
          </span>)
      }    
    </Wrapper>
```

Reply:
Done!